cloud/deploys/bin/azure/key-vault-setup

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env bash
+source lib/out.sh
+source azure/key-vault.sh
+
+# DOC: Create key vault instance, set permissions, and generate and set secrets
+
+location="EastUS"
```

Comment:
suggestion (non-blocking): my guess would be this would be best as a required arg to encourage explicit use, fine leaving as is for now if you'd prefer though

Reply:
Updated to be a required arg